crates/freeze/src/datasets/logs.rs

```diff
@@ -1,7 +1,8 @@
 use std::{collections::HashMap, sync::Arc};
 
 use ethers::prelude::*;
-use polars::prelude::*;
+use ethers_core::abi::{AbiEncode, HumanReadableParser, RawLog, Token};
+use polars::{export::ahash::HashSet, prelude::*};
```

Comment:
why import `HashSet` from polars instead of std?

Reply:
auto import, will fix